boxtree/tree.py

```diff
@@ -111,6 +111,83 @@ class box_flags_enum(Enum):  # noqa
 # }}}
 
 
+# {{{ tree of boxes
+
+class TreeOfBoxes:
```

Comment:
Given that the attributes don't seem to agree with the format established by `Tree`, document what they actually are.

Reply:
d636ec9b1b380890631e21a4ab3df33867c98e67